features/cmake/find_package/exported_targets_multiconfig/build.bat

```diff
@@ -0,0 +1,23 @@
+pushd hello
```

Comment:
Why are we dealing with .bat and .sh scripts instead of python scripts? That would avoid some repetition.

Reply:
That's not a question for this PR, but for the whole repository. I followed the de-facto convention.

Reply:
For me, it is not so bad. No python magic at all, just run the script. Just a bit inconvenient for us to provide two scripts.